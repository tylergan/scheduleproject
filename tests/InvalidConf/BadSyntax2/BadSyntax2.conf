on at run /bin/date