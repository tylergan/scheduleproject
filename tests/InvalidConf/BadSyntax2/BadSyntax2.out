error in configuration: on at run /bin/date
